docs/preview/connected-registry/README.md

```diff
@@ -0,0 +1,63 @@
+# ACR connected registry (Private Preview) instructions
+
+This article provides a guidance for use of the connected registry feature of Azure Container Registry (ACR) during the limited preview. 
+
+To request preview access, submit your contact details using this [form](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR1OsLxas9SdIhfyFenqqkolUMkFKMTdDSU45SFQzU0o0WUNROVAySkRINy4u) and we will get in touch with you.
+
+## Available Regions
+
+During limited preview period, the connected registry functionality is available in dedicated stamps in the following Azure regions:
+
```

Comment:
Should this list be the same as the list on the request form?

Reply:
Yeah, I will update the request form